tests/test_websockets.py

```diff
@@ -0,0 +1,150 @@
+import pytest
+import asyncio
+import websockets
+
+
+@pytest.mark.asyncio
+async def test_websocket_route(liveclient):
+    ev = asyncio.Event()
+
+    @liveclient.app.route('/ws', protocol="websocket")
+    async def handler(request, ws, **params):
+        assert ws.subprotocol is None
+        ev.set()
+
+    with liveclient as query:
+        response = await query('GET', '/ws', headers={
+            'Upgrade': 'websocket',
+            'Connection': 'upgrade',
+            'Sec-WebSocket-Key': 'hojIvDoHedBucveephosh8==',
+            'Sec-WebSocket-Version': '13'})
+
+    assert ev.is_set()
+    assert response.status == 101
```

Comment:
Can you use `http.HTTPStatus.SWITCHING_PROTOCOLS` here and there for consistency?

Reply:
Done